src/interface/stmt.ml

```diff
@@ -167,9 +166,9 @@ module type Logic = sig
   val assert_     : ?loc:location -> term -> t
   (** Add an assertion to the current set of assertions. *)
 
-  val check_sat   : ?loc:location -> term list -> t
+  val check_sat   : ?loc:location -> ?name:id -> term list -> term list -> t
```

Comment:
there's no need to take two term lists as you never use the second one in this PR. If we ever need the second list, then we'll add another function.